lib/servactory/inputs/input.rb

```diff
@@ -155,6 +161,29 @@ def add_collection_option_with(type, options) # rubocop:disable Metrics/MethodLe
         )
       end
 
+      def add_object_option_with(type, options) # rubocop:disable Metrics/MethodLength
+        collection_of_options << Servactory::Maintenance::Attributes::Option.new(
+          name: :schema,
+          attribute: self,
+          validation_class: Servactory::Inputs::Validations::Type,
+          define_methods: [
+            Servactory::Maintenance::Attributes::DefineMethod.new(
+              name: :object_mode?,
+              content: ->(**) { object_mode_class_names.include?(type) }
+            )
+          ],
+          define_conflicts: [
+            Servactory::Maintenance::Attributes::DefineConflict.new(
+              content: -> { :object_vs_inclusion if object_mode? && inclusion_present? }
```

Comment:
Need to add a conflict check for the collection. And the same for the collection about the object.